x-pack/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.test.ts

```diff
@@ -68,6 +69,11 @@ describe('rules_notification_alert_type', () => {
     modulesProvider: jest.fn(),
     resultsServiceProvider: jest.fn(),
   };
+  const listMock = {
```

Comment:
This is something that would be nice to have as part of the lists plugin itself: https://github.com/elastic/kibana/blob/master/src/core/MIGRATION.md#writing-mocks-for-your-plugin